script/cibuild.sh

```diff
@@ -47,6 +47,13 @@ export GITTEST_REMOTE_URL="git://localhost/test.git"
 # Run the test suite
 ctest -V -R libgit2_clar || exit $?
 
+# Run the standalone fuzz targets
+for fuzz_target in $(find ./fuzz/* -type f -executable); do
+	fuzz_target_name=$(basename "${fuzz_target}")
+	# TODO: Make this exit on failures once all fixes are in place.
+	"${fuzz_target}" "../fuzz/corpora/${fuzz_target_name}"
+done
```

Comment:
Cool, that's a good thing to have. I guess making it work on AppVeyor wouldn't make any sense at all, would it?

Reply:
maybe, i can give it a try later :)